electron/main.ts

```diff
@@ -1,4 +1,6 @@
 import * as path from 'path'
+//import { EventEmitter } from 'events'; EventEmitter.defaultMaxListeners = 20
```

Comment:
This commented line will be use or can be removed?

Reply:
I am unsure, when I was testing, electron wasn't creating a window and all I saw was that the maxlisteners was exceeded, but it still does this. I'll need to look into it further as it just wasn't launching for me when I used the custom transports.

Reply:
Removed this 👍🏻